### PR TITLE
Adds GHDL v0.33

### PR DESCRIPTION
**Pull request by larsschwegmann**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/ghdl.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'ghdl' do
+  version '0.33'
+  sha256 '665f487c936be041d27a0f9d638109e63f1706cf7b3d8ceddb81db5fb2f71d0e'
+
+  url "https://github.com/tgingold/ghdl/releases/download/v#{version}/ghdl-#{version}-llvm-darwin13.pkg"
```
Comment (vitorgalvao) [inline Casks/ghdl.rb]: Needs a `# github.com/tgingold/ghdl was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by larsschwegmann** [comment]
@vitorgalvao Updated my Cask. Please review.

**Comment by vitorgalvao** [comment]
https://github.com/caskroom/homebrew-cask/issues/14058

**Comment by vitorgalvao** [comment]
Looking good. Merging after Travis passes.

**Pull request merged**